src/layman/layer/util.py

```diff
@@ -262,34 +263,96 @@ def get_same_or_missing_prop_names(workspace, layername):
             'found': {
                 'db_connection': connection_string,
-                'schema': connection.scheme,
+                'scheme': connection.scheme,
             }
         })
 
-    params = parse.parse_qs(connection.query)
-    url = connection._replace(query='').geturl()
-    result = ConnectionString(url=url,
-                              table=params.get('table', [None])[0],
-                              geo_column=params.get('geo_column', [None])[0],
-                              )
-    if not all([result.url, result.table, result.geo_column]):
+    query = parse.parse_qs(connection.query)
+    table = query.pop('table', [None])[0]
+    geo_column = query.pop('geo_column', [None])[0]
+    connection = connection._replace(query=parse.urlencode(query, True))
+    uri = parse.urlunparse(connection)
+    if not all([uri, table, geo_column, connection.netloc]):
+        raise LaymanError(2, {
+            'parameter': 'db_connection',
+            'message': 'Parameter `db_connection` is expected to have `url` part, `netloc` part and `table` and `geo_column` query parameters',
+            'expected': 'postgresql://<username>:<password>@<host>:<port>/<dbname>?table=<schema>.<table_name>&geo_column=<geo_column_name>',
+            'found': {
+                'db_connection': connection_string,
+                'url': uri,
+                'netloc': connection.netloc,
+                'table': table,
+                'geo_column': geo_column,
+            }
+        })
+
+    schema, table = table.split('.')
+    if not table:
         raise LaymanError(2, {
             'parameter': 'db_connection',
-            'message': 'Parameter `db_connection` is expected to have `url` part and `table` and `geo_column` query parameters',
-            'expected': 'postgresql://<username>:<password>@<host>:<port>/<dbname>?table=<table_name>&geo_column=<geo_column_name>',
+            'message': 'Parameter `db_connection` is expected to have query parameter `table` in format `<schema>.<table_name>`',
+            'expected': 'postgresql://<username>:<password>@<host>:<port>/<dbname>?table=<schema>.<table_name>&geo_column=<geo_column_name>',
             'found': {
                 'db_connection': connection_string,
-                'url': result.url,
-                'table': result.table,
-                'geo_column': result.geo_column,
+                'table': table,
             }
         })
 
+    try:
+        conn_cur = db_util.create_connection_cursor(uri, encapsulate_exception=False)
+    except psycopg2.OperationalError as exc:
+        raise LaymanError(2, {
+            'parameter': 'db_connection',
+            'message': 'Invalid postgres connection string. Not able to connect.',
+            'expected': 'postgresql://<username>:<password>@<host>:<port>/<dbname>?table=<schema>.<table_name>&geo_column=<geo_column_name>',
```

Comment:
Encapsulate value `postgresql://<username>:<password>@<host>:<port>/<dbname>?table=<schema>.<table_name>&geo_column=<geo_column_name>` to global constant, probably defined in layman.layer.util